lib/tasks/audit_code.rake

```diff
@@ -478,6 +467,43 @@ Usage:
     flag_file_as_safe(release, ENV['reviewed_by'], ENV['comments'], ENV['file'])
   end
 
+  desc "Review safety of selected source file.
+Usage:
+  Review a source file:   #{rake_cmd} audit:review file=f [comments=...]"
+  task(:review) do
+    fname = ENV['file']
+    if fname.nil?
```

Comment:
Unless I'm missing something, this still requires you to run `rake audit:review` for each file individually, which would seem to defeat the point a little?

Reply:
With existing tasks, you need to run the following command to flag a file safe, and the fact of long hash value force you to copy and paste the code.
`bundle exec rake audit:safe release=HASH_VALUE file=FILE_NAME reviewed_by=USER comments=""`

the new task require you to run the code `bundle exec rake audit:review file=FILE_NAME`, where file name can be completed by pressing TAB key, and you can then review and flag the file safe or not using the command prompt.

The main difference is to save you from repeated copy and paste.

Reply:
This just isn't doing what I intended. Rather than tweak this PR further I suggest we look at this again.